bluemira/utilities/tools.py

```diff
@@ -299,6 +300,15 @@ def cross(
 # =====================================================
 
 
+def floatify(x: npt.ArrayLike):
+    """
+    Avoid numpy warnings for float(x) for >0 rank scalars
+    """
+    if x is None:
+        raise TypeError("argument must be a string or a real number, not 'NoneType'")
+    return np.asarray(x, dtype=float).item()
```

Comment:
```suggestion
    return np.asarray(x, dtype=float).item(0)
```

I think this is more explicit

Reply:
I'm not going to do this because it will then not error when converting a multi element array to a float (which we want if we're emulating the previous behaviour). I will add this to the docstring.